WcaOnRails/config/locales/en.yml

```diff
@@ -236,7 +236,17 @@ en:
         competition_events: "Events"
         enable_donations: "I want to enable donations additionally to collecting the registration fees"
         championship_type: "Championship type"
-        registration_requirements: "Registration requirements"
+        registration_requirements: "Extra registration requirements"
+        free_competition_enabled: "Registering for this competition is free"
+        on_the_spot_registration_enabled: "I want to accept on the spot registrations"
+        on_the_spot_entry_fee: "I want to use a different entry fee for on the spot registrations"
+        on_the_spot_entry_fee_lowest_denomination: "On the spot entry fee"
```

Comment:
Out of curiosity, how much thought has gone into the phrase "on the spot" vs "at the door"? I feel like I've usually heard the term "at the door".

Reply:
I think "on the spot", "walk in", and "at the door" are roughly equally used around the world.